docs/versions/latest/Release.md

```diff
@@ -1,34 +1,37 @@
-[//]: # (VERSION_PLACEHOLDER DO NOT DELETE)
-[//]: # (Used when working on a new release. Placed together with the Version.md)
-[//]: # (Nothing here is optional. If a step must not be performed, it must be said so)
-[//]: # (Do not fill the version, it will be done automatically)
-[//]: # (Quick Intro to what is the focus of this release)
+[//]: # "VERSION_PLACEHOLDER DO NOT DELETE"
+[//]: # "Used when working on a new release. Placed together with the Version.md"
+[//]: # "Nothing here is optional. If a step must not be performed, it must be said so"
+[//]: # "Do not fill the version, it will be done automatically"
+[//]: # "Quick Intro to what is the focus of this release"
 
 ## Breaking Changes
 
-[//]: # (### *Breaking Change*)
-[//]: # (Describe the breaking change AND explain how to resolve it)
-[//]: # (You can utilize internal links /e.g. link to the upgrade procedure, link to the improvement|deprecation that introduced this/)
+[//]: # "### *Breaking Change*"
+[//]: # "Describe the breaking change AND explain how to resolve it"
+[//]: # "You can utilize internal links /e.g. link to the upgrade procedure, link to the improvement|deprecation that introduced this/"
 
 ## Deprecations
 
-[//]: # (### *Deprecation*)
-[//]: # (Explain what is deprecated and suggest alternatives)
-
-[//]: # (Features -> New Functionality)
+[//]: # "### *Deprecation*"
+[//]: # "Explain what is deprecated and suggest alternatives"
+[//]: # "Features -> New Functionality"
 
 ## Features
 
-[//]: # (### *Feature Name*)
-[//]: # (Describe the feature)
-[//]: # (Optional But higlhy recommended Specify *NONE* if missing)
-[//]: # (#### Relevant Documentation:)
+[//]: # "### *Feature Name*"
+[//]: # "Describe the feature"
+[//]: # "Optional But higlhy recommended Specify *NONE* if missing"
+[//]: # "#### Relevant Documentation:"
+
+### Support of Objects in the VROES.Shims.arrayFrom() Method
+
+Add support for objects in the `VROES.Shims.arrayFrom()` method so its behavior is similar to the standard `Array.from()` method.
 
 ### Support python 3.10 runtime
 
 Add support for python 3.10 runtime in Orchestrator. This is now the default, since python 3.7 is deprecated.
 
-### *New `@ActionItem` decorator for Workflows*
+### _New `@ActionItem` decorator for Workflows_
```

Comment:
This too

Reply:
Discussed with @VenelinBakalov , no need for reverting currently.